src/Assert.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Laminas\Mvc\View;
+
+use Laminas\Mvc\View\Exception\AssertionFailedException;
+use Webmozart\Assert\Assert as WebmozartAssert;
+
+/** @internal  */
+final class Assert extends WebmozartAssert
+{
+    /**
+     * @param string $message
+     * @throws AssertionFailedException
+     * @psalm-pure
+     * @psalm-return never
```

Comment:
not 100% sure but isnt "never" only to be used when `exit` or `die` is somewhere within the method?

Reply:
This method is guaranteed to throw, so it never returns. Because it's annotated this way in `webmozart/assert`, tbh, I just duplicated the type.